jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level = INFO
+io.netty.level = INFO
+io.gcr.level = INFO
```

Comment:
I added it because netty logs from background threads.  However there is still the issue that not all of netty's log mechanism appears not to see these settings anyway, so perhaps we just need an explicit filter in the publish method?